src/kani-verifier/src/lib.rs

```diff
@@ -58,6 +58,7 @@ pub fn proxy(bin: &str) -> Result<()> {
             setup(None)
         }
     } else {
+        fail_if_in_dev_environment()?;
```

Comment:
How are we supposed to test this crate locally?

Reply:
We don't except by using the Dockerfile.